Woof/Woof/ViewModel/SitterProfileViewModel.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+/// The view model for a pet sitter profile view is responsible for receiving and processing user actions
+/// to change model data and passes back the updated model data.
+final class SitterProfileViewModel: ObservableObject {
+    // MARK: - Internal interface
+
+    /// The name of the pet sitter.
+    @Published var name: String = ""
+
+    /// The surname of the pet sitter.
+    @Published var surname: String = ""
+
+    /// The phone of the pet sitter.
+    @Published var phone: String = ""
+
+    /// The additional information about the pet sitter, like his experience with dogs, favourite places for walks,
+    /// special skills, certificates, etc.
+    @Published var bio: String = ""
+
+    /// The price per hour for walking charged by the pet sitter.
+    @Published var pricePerHour: Double = 0
```

Comment:
Why we use `Double`? Should we convert this value to suitable for representation? String for example.

Reply:
For feature we need Doodle type. I don't see a problem with using `Double`.

Reply:
Will we convert `Double` to `String` to output in a view? Why don't we prepare the data in the view model?

Reply:
Thanks, fixed it.